### DISCUSSION TITLE
Thoughts about the level of announcements from the LiveAnnouncer?

### DISCUSSION THREAD

**Discussion by heisand** [adobe/react-spectrum#4909] [Q&A] [answered]
I have a little question about the LiveAnnouncer, here related to the comboBox. When I have an input field which triggers opening of the list box on focus, the announcer then reads "X suggestions found" before there is time to announce what kind of element we are in. (I my case I have a list which already has a set of options that show before I write anything in the input). In other words, it first reads how many suggestions we have, then that we are in a combobox with a list. 

This sounds a little bit weird (and would in my mind make more sense the other way around). Have you made any thoughts on the level of this announcement, and perhaps setting it as `polite` instead of `assertive`? Or at least doing it somewhat configurable?

**Comment by LFDanLu** [comment]
Interesting, when I test our ComboBox in [this story](https://reactspectrum.blob.core.windows.net/reactspectrum/d1a30201a11db8fea01dd09ce422bbe45c4dce5a/storybook/index.html?path=/story/combobox--all-controlled&args=menuTrigger:focus&providerSwitcher-express=false) in VoiceOver on Safari it announces the ComboBox via "menu popup, combobox" before reading "6 suggestions found" after I tab on to it (note I deleted the input value first before testing to get the full list of items). Same result when testing this React Aria implementation as well: https://codesandbox.io/s/sad-sun-2wj6yk Do you have a base reproduction I can take a look at?

**Reply by heisand** [reply]
I will add a screen recording to illustrate what I see. I navigate using ctrl+option+arrows (with VoiceOver on MacOS in Chrome), and I navigate rather quickly after opening the page. Then in this case, it is actually announced "6 alternatives found" three times. After that, it is announced that I am in currently in a ComboBox.


https://github.com/adobe/react-spectrum/assets/15145686/4d880444-f02e-40d3-8339-53c221e6f2e3

**Reply by LFDanLu** [reply, marked as answer]
Thanks for the reproduction video, I've confirmed the same when using control+option+arrow keys instead of using Tab to navigate to the combobox. It seems to happen less often when using Tab to focus the combobox, but happens all the time when using control+option+arrow keys. I've tried changing the announcement to be `polite` locally but it seems to still be a problem interestingly enough. We can look more into it though, will convert this into an issue
